core/chains/evm/config/toml/config.go

```diff
@@ -920,6 +921,19 @@ func (p *NodePool) setFrom(f *NodePool) {
 	p.Errors.setFrom(&f.Errors)
 }
 
+func (p *NodePool) ValidateConfig(finalityTagEnabled *bool) (err error) {
+	if finalityTagEnabled != nil && *finalityTagEnabled {
+		if p.FinalizedBlockPollInterval == nil {
+			err = multierr.Append(err, commonconfig.ErrMissing{Name: "FinalizedBlockPollInterval", Msg: "required when FinalityTagEnabled is true"})
+		}
+		if p.FinalizedBlockPollInterval.Duration() <= 0 {
```

Comment:
Leads to panic if FinalizedBlockPollInterval is nil

Reply:
Will return error if nil